lib/types/cql-utils.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const types = require("./index");
+const rust = require("../../index");
+const { ResponseError } = require("../errors");
+const TimeUuid = require("./time-uuid");
+const Uuid = require("./uuid");
+const Duration = require("./duration");
+
+/**
+ * Wrap value into QueryParameterWrapper based on the type
+ * @param {CqlType} type
+ * @param {*} value
+ * @returns {rust.QueryParameterWrapper}
+ */
+function getWrapped(type, value) {
+    switch (type) {
+        case rust.CqlType.Ascii:
+            return rust.QueryParameterWrapper.fromAscii(value);
+        case rust.CqlType.Blob:
+            return rust.QueryParameterWrapper.fromBlob(value);
+        case rust.CqlType.Boolean:
+            return rust.QueryParameterWrapper.fromBoolean(value);
+        case rust.CqlType.Counter:
+            return rust.QueryParameterWrapper.fromCounter(value);
+        case rust.CqlType.Double:
+            return rust.QueryParameterWrapper.fromDouble(value);
+        case rust.CqlType.Duration:
+            if (!(value instanceof Duration)) throw new TypeError("");
+            return rust.QueryParameterWrapper.fromDuration(value.getInternal());
```

Comment:
Why is this check present only for Duration?

Reply:
Well, and for Uuid and TimeUuid.

Reply:
For those types we call internal function on the passed object. For other types, the program will throw an error with meaningful description, while it's not the case for those types.

(I've updated the error messages for those TypeErrors)

Reply:
Could you put the above explanation in a comment?